packages/@aws-cdk/aws-iot-actions/README.md

```diff
@@ -49,6 +51,46 @@ new iot.TopicRule(this, 'TopicRule', {
 });
 ```
 
+## Put objects to a S3 bucket
+
+The code snippet below creates an AWS IoT Rule that put objects to a S3 bucket
+when it is triggered.
+
+```ts
+import * as iot from '@aws-cdk/aws-iot';
+import * as actions from '@aws-cdk/aws-iot-actions';
+import * as s3 from '@aws-cdk/aws-s3';
+
+const bucket = new s3.Bucket(this, 'MyBucket');
+
+new iot.TopicRule(this, 'TopicRule', {
+  sql: iot.IotSql.fromStringAsVer20160323("SELECT topic(2) as device_id FROM 'device/+/data'"),
+  actions: [new actions.S3PutObjectAction(bucket)],
+});
+```
+
+The property `key` of `S3PutObjectAction` is given the value `${topic()}/${timestamp()}` by default. This `${topic()}`
+and `${timestamp()}` is called Substitution templates. For more information see 
+[this documentation](https://docs.aws.amazon.com/iot/latest/developerguide/iot-substitution-templates.html).
+In above sample, `${topic()}` is replaced by a given MQTT topic as `device/001/data`. And `${timestamp()}` is replaced
+by the number of the current timestamp in milliseconds as `1636289461203`. So if the MQTT broker receives an MQTT topic
+`device/001/data` on `2021-11-07T00:00:00.000Z`, the S3 bucket object will be put to `device/001/data/1636243200000`. 
+
+You can also set specific `key` as following:
+
+```ts
+new iot.TopicRule(this, 'TopicRule', {
+  sql: iot.IotSql.fromStringAsVer20160323(
+    "SELECT topic(2) as device_id, year, month, day FROM 'device/+/data'",
+  ),
+  actions: [
+    new actions.S3PutObjectAction(bucket, {
+      key: '${year}/${month}/${day}/${topic(2)}',
```

Comment:
Show `cannedAcl` too?

Reply:
OK, I see below. In the future, I would merge these examples, because otherwise the ReadMe becomes way too long.